spec/services/ccms/case_add_requestor_xml_blocks_spec.rb

```diff
@@ -716,39 +600,19 @@ module CCMS # rubocop:disable Metrics/ModuleLength
             attributes.each do |entity_attribute_pair|
               entity, attribute = entity_attribute_pair
               block = XmlExtractor.call(xml, entity, attribute)
-              expect(block).to be_present
-              expect(block).to have_response_type 'boolean'
-              expect(block).to have_response_value 'false'
+              expect(block).to have_boolean_response false
             end
           end
         end
 
         it 'CATEGORY_OF_LAW should be hard coded to FAMILY' do
           block = XmlExtractor.call(xml, :global_means, 'CATEGORY_OF_LAW')
-          expect(block).to be_present
-          expect(block).to have_response_type 'text'
-          expect(block).to have_response_value 'FAMILY'
+          expect(block).to have_text_response 'FAMILY'
         end
 
         it 'CASES_FEES_DISTRIBUTED should be hard coded to 1' do
           block = XmlExtractor.call(xml, :global_merits, 'CASES_FEES_DISTRIBUTED')
-          expect(block).to be_present
-          expect(block).to have_response_type 'number'
-          expect(block).to have_response_value '1'
-        end
-
-        it 'LEVEL_OF_SERVICE should be hard coded to 3' do
-          attributes = [
-            [:proceeding_merits, 'LEVEL_OF_SERVICE'],
-            [:proceeding, 'LEVEL_OF_SERVICE']
-          ]
-          attributes.each do |entity_attribute_pair|
-            entity, attribute = entity_attribute_pair
-            block = XmlExtractor.call(xml, entity, attribute)
-            expect(block).to be_present
-            expect(block).to have_response_type 'text'
-            expect(block).to have_response_value '3'
-          end
+          expect(block).to have_number_response 1
         end
 
         it 'CLIENT_INVOLVEMENT_TYPE should be hard coded to A' do
```

Comment:
This test seems to be a duplicate of the one on line 558.

Reply:
This also applies to the two following tests, `'NEW_APPL_OR_AMENDMENT should be hard coded to APPLICATION'` and `'USER_TYPE should be hard coded to EXTERNAL'` which are duplicated above.